src/components/App/App.js

```diff
@@ -36,24 +34,22 @@ function App() {
   }, [])
   
   return (
-    <>
+    <UserContext.Provider value={[currentUser, setCurrentUser]}>
```

Comment:
`value = {{currentUser, setCurrentUser}}`

Reply:
oh, maybe this is why it worked? I'll try it with this additional line and see if it works. That way we don't have an unused state.

Reply:
const [currentUser, setCurrentUser] = useContext(UserContext);   -if we wanted the object approach

const setCurrentUser = useContext(UserContext)[1];  - array approach